MediaBrowser.Api/Images/RemoteImageService.cs

```diff
@@ -261,27 +260,26 @@ public async Task<object> Get(GetRemoteImage request)
         /// <returns>Task.</returns>
         private async Task DownloadImage(string url, Guid urlHash, string pointerCachePath)
         {
-            using (var result = await _httpClient.GetResponse(new HttpRequestOptions
+            using var result = await _httpClient.GetResponse(new HttpRequestOptions
             {
                 Url = url,
-                BufferContent = false
+                BufferContent = false,
+            }).ConfigureAwait(false);
 
-            }).ConfigureAwait(false))
-            {
-                var ext = result.ContentType.Split('/').Last();
+            var ext = result.ContentType.Split('/').Last();
 
-                var fullCachePath = GetFullCachePath(urlHash + "." + ext);
+            var fullCachePath = GetFullCachePath(urlHash + "." + ext);
 
-                Directory.CreateDirectory(Path.GetDirectoryName(fullCachePath));
-                using (var stream = result.Content)
-                using (var filestream = new FileStream(fullCachePath, FileMode.Create, FileAccess.Write, FileShare.Read, IODefaults.FileStreamBufferSize, true))
-                {
-                    await stream.CopyToAsync(filestream).ConfigureAwait(false);
-                }
+            Directory.CreateDirectory(Path.GetDirectoryName(fullCachePath));
 
-                Directory.CreateDirectory(Path.GetDirectoryName(pointerCachePath));
-                File.WriteAllText(pointerCachePath, fullCachePath);
+            await using (var stream = result.Content)
+            {
+                await using var fileStream = new FileStream(fullCachePath, FileMode.Create, FileAccess.Write, FileShare.Read, IODefaults.FileStreamBufferSize, true);
```

Comment:
Why is this `using var fileStream` pulled inside the block here? AFAIK the previous method having it outside the block is equivalent but IMO easier to read

Reply:
done